share/core.go

```diff
@@ -0,0 +1,315 @@
+// Package share implements different secret sharing mechanisms, including
+// simple Shamir secret sharing, verifiable secret sharing (vss), and public
+// verifiable secret sharing (pvss).
+package share
+
+import (
+	"crypto/cipher"
+	"errors"
+
+	"github.com/dedis/crypto/abstract"
+)
+
+// This file provides the core functionality for secret sharing including
+// Shamir's scheme and public commitment polynomials.
```

Comment:
this comment should be at the top level also, because it is not shown otherwise in the godoc (IIRC)